drivers/soundwire/cadence_master.c

```diff
@@ -836,6 +836,7 @@ EXPORT_SYMBOL(sdw_cdns_exit_reset);
  */
 int sdw_cdns_enable_interrupt(struct sdw_cdns *cdns, bool state)
 {
+	u32 slave_state0, slave_state1;
```

Comment:
you could place these under the `if` where they are actually used

Reply:
yes, I will refine it.